### PR TITLE
fix(api): incorrect rtc when joining ordinary room

### PR DESCRIPTION
**Pull request by crimx**
New RTC is wrongly generated for studuent when joining the same room subsequently.

### CODE REVIEW

**Review by hyrious — APPROVED** [review]

File: src/v1/controller/room/join/Ordinary.ts [hunk 1]
```diff
@@ -33,16 +33,12 @@ export const joinOrdinary = async (roomUUID: string, userUUID: string): Response
...
-
-        if (roomUserInfo === undefined) {
-            throw new Error("When getting the rtcUID of the owner, the result does not exist");
-        }
+    const roomUserInfo = await RoomUserDAO().findOne(["rtc_uid"], {
+        room_uuid: roomUUID,
+        user_uuid: userUUID,
+    });
 
+    if (roomUserInfo?.rtc_uid) {
```
Comment (BlackHole1) [inline src/v1/controller/room/join/Ordinary.ts:41]: 这里改一下

Comment (BlackHole1) [inline src/v1/controller/room/join/Ordinary.ts:41]: 这里为了保持统一，改成  `roomUserInfo !== undefined` 吧

**Review by BlackHole1 — CHANGES_REQUESTED** [review]
A concern was raised

File: src/v1/controller/room/join/Ordinary.ts (hunk 1 above)
Comment (crimx) [reply src/v1/controller/room/join/Ordinary.ts]: 已修改

**Pull request merged**